scripts/analysis.py

```diff
@@ -638,50 +583,171 @@ def final_stockpile(cursor, facility):
     return outstring
 
 
-    return outstring
+def fuel_into_reactors(cursor):
+    """ Finds timeseries of mass of fuel received by reactors
 
- 
-def fuel_usage_timeseries(cursor, fuel_list):
-    """ Calculates total fuel usage over time
+    Parameters
+    ----------
+    cursor: sqlite cursor
+        sqlite cursor
+
+    Returns
+    -------
+    get_timeseries: function
```

Comment:
This does not return the get_timeseries function (That's possible, but it's not what you're doing). It returns the get_timeseries function on a particular cursor with various parameters derived from the cursor...